src/BootstrapMixin.js

```diff
@@ -10,9 +11,12 @@ const BootstrapMixin = {
     bsClass: CustomPropTypes.keyOf(styleMaps.CLASSES),
     /**
      * Style variants
+     *
+     * You can set custom style classes
+     * http://localhost:4000/components.html#custom-styles
```

Comment:
Whoops :smile:  Sorry.
Actually I was going to ask what is the best way to include working link to another section here ?
Of course it should be like this http://react-bootstrap.github.io/components.html#custom-styles